### ISSUE TITLE
[Feature Request] Parameter permutation syntax

### ISSUE THREAD

**Issue by Radtoo** [derrian-distro/LoRA_Easy_Training_Scripts#64]
If we could use something like [5,10,20] on fields like number of repeats and [1e-04,1e-05,5e-05] on learning rate and easy training scripts would run all the permutations, that would be quite expedient with how many variants have to be tried in some cases.

Maybe the output folders + output filenames would be suffixed with _00001 _00002 and so on. Perhaps with a .toml.diff inside to know which permutation settings each folder represents.

**Comment by derrian-distro** [comment]
unfortunately, that would introduce too many things to check for for me to really do any error checking on it. I'm probably not going to implement this one.

**Comment by Radtoo** [comment]
Unfortunate that it is not easy to do this. Thanks for giving it your consideration!
